src/components/Form.jsx

```diff
@@ -1,6 +1,9 @@
 import React, { useState, useRef } from 'react';
+import { NavLink } from 'react-router-dom';
 import axios from 'axios';
 
+const link = (path, text) => <NavLink to={path} exact activeClassName="active" className="link">{text}</NavLink>;
```

Comment:
you may add another parameter to the function `link`: if its defined, use it for the class name of your link. if it is undefined, use 'link' as a fallback:

    const link = (path, text, className) => <NavLink to={path} exact activeClassName="active" className={className ?? 'link'}>{text}</NavLink>;